components/resources/library/src/commonMain/kotlin/org/jetbrains/compose/resources/ResourceEnvironment.kt

```diff
@@ -5,11 +5,12 @@ import androidx.compose.runtime.*
 import androidx.compose.ui.platform.LocalDensity
 import androidx.compose.ui.text.intl.Locale
 
-internal data class ResourceEnvironment(
-    val language: LanguageQualifier,
-    val region: RegionQualifier,
-    val theme: ThemeQualifier,
-    val density: DensityQualifier
+@ExperimentalResourceApi
+data class ResourceEnvironment internal constructor(
```

Comment:
[Don't use data classes in an API﻿](https://kotlinlang.org/docs/jvm-api-guidelines-backward-compatibility.html#don-t-use-data-classes-in-an-api)